CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+<h1 style="border-bottom: 0;">Contribution Guideline</h1>
```

Comment:
```suggestion
# Contribution Guideline
```

why not just simply this? (this comment applies to all files)

Reply:
I just thought that it looked better without the intial border:
With border:
<img width="1233" alt="image" src="https://github.com/user-attachments/assets/ccb99bc2-5429-43b7-94da-0135143a9333" />
Without border:
<img width="1276" alt="image" src="https://github.com/user-attachments/assets/23b7c576-0a7f-4cad-b1f3-56be5610080d" />

What do you think?

Reply:
I wouldn't mess up with github styling decision. Let's just use plain Markdown, these documents may be read in editors for example